community/index.md

```diff
@@ -120,6 +120,16 @@ See our [events page](/events/).
 Some local Scala user groups are listed on
 [Meetup](https://www.meetup.com/topics/scala/).
 
+ - [London Scala User Group](https://www.meetup.com/london-scala/)
+ - [Warsaw Scala Enthusiasts](https://www.meetup.com/ScalaWAW/)
+ - [Kraków Scala User Group](https://www.meetup.com/Krakow-Scala-User-Group/)
+ - [Wrocław Scala User Group](https://www.meetup.com/wroclaw-scala-user-group/)
+ - [Bay Area Scala](https://lu.ma/scala)
+ - [SF Scala](http://www.meetup.com/SF-Scala/)
+ - [Scala Bay](https://www.meetup.com/scala-bay/)
```

Comment:
@zainab-ali Thank you so much for adding this. Super appreciated. You are awesome.